compiler-rt/lib/scudo/standalone/tests/common_test.cpp

```diff
@@ -50,7 +50,7 @@ TEST(ScudoCommonTest, SKIP_ON_FUCHSIA(ResidentMemorySize)) {
   memset(P, 1, Size);
   EXPECT_GT(getResidentMemorySize(), OnStart + Size - Threshold);
 
-  MemMap.unmap(MemMap.getBase(), Size);
```

Comment:
I presume that the usage of Size here was incorrect and this was supposed to be a clean-up, correct?

Reply:
Oh yes, the `Size` is always equal to `getCapacity()` here.